File: 1-first-year-2021-2022/Programming-and-laboratory/Exercises/01_Gennaio/2022.01.22/050_ControlloDiFlusso/sommaInteri.c
// Si scriva un programma che stampi la somma dei primi n numeri naturali, dove n è un numero richiesto all’utente. //
#include <stdio.h>

int main()
{
    int naturale;
    int somma = 0;

    printf("Inserisci un numero naturale, ti indichero' la somma di tutti i valori precedenti (ed incluso) al numero da te inserito.\n");
    scanf("%d", &naturale);

    for (int i = 0; i <= naturale; i++)
    {
        somma += i;
    }
    printf("La somma dei valori pari ed inferiori a %d, numero naturale da te indicato, e' pari a %d.\n", naturale, somma);
}

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/sommaInteri/sommaInteri.java
import java.util.*;

public class sommaInteri {
    public static void main(String args[]){
        int num = 0;
        int somma = 0;
        int ciclo = 1;

        Scanner console = new Scanner(System.in);

        while (ciclo <= 2){
            System.out.print("Inserisci il numero " + ciclo + ": ");
            num = console.nextInt();
            somma += num;
            ciclo++;
        }

        console.close();

        System.out.println("La somma dei valori da te inseriti e' pari a " + somma + ".");
    }
}